studio.ps1

```diff
@@ -67,10 +67,24 @@ function Stage-Three {
     $kitArgs = "/q"
 
     if ($KITTYPE -eq "HLK") {
-        if (Test-Path -Path "$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe") {
-            $kitPath = "$PSScriptRoot\Kits\HLK${HLKKITVER}\HLKSetup.exe"
-        } else {
-            $kitPath = "$PSScriptRoot\Kits\HLK${HLKKITVER}Setup.exe"
+        $mountedDisks = Get-WmiObject -Class Win32_LogicalDisk | Where-Object { $_.DriveType -eq 5 }
+        $foundHCK = $false
+
+        foreach ($disk in $mountedDisks) {
+            $hckSetupPath = Join-Path -Path $disk.DeviceID -ChildPath "HLKSetup.exe"

```

Comment:
It's an odd naming, considering that this code is executed when `$KITTYPE -eq "HLK"` instead of `HCK`.
It also does not distinguish the variable from `kitPath` well. What about `kitPathInDisk`?

Reply:
I suggested renaming `hckSetupPath` to `kitPathInDisk` because it's a path in `$disk`. Renaming `kitPath` to `kitPathInDisk` makes no sense.

Reply:
> It's an odd naming, considering that this code is executed when `$KITTYPE -eq "HLK"` instead of `HCK`.

About  "HLK" and  "HCK" naming, it was there. I not sure that i need to touch this if it's works.